src/toolbar.scss

```diff
@@ -0,0 +1,170 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-toolbar;
+$toolbar-height-compact: 2rem;
+$toolbar-height-cozy: 2.75rem;
+
+$toolbar-separator-height-compact: 1.5rem;
+$toolbar-separator-height-cozy: 2rem;
+
+$toolbar-spacing: 0.5rem;
+$toolbar-padding: 0 0.5rem;
+$title-toolbar-padding: 0 0.5rem 0 0.1rem;
+$title-toolbar-height: 2.75rem;
+
+%toolbarStandardColors {
+  @include fd-hover() {
+    background-color: var(--sapList_Hover_Background);
+  }
+
+  @include fd-active() {
+    background-color: var(--sapActiveColor);
+  }
```

Comment:
if you put these values in `fd-toolbar` (You don't need --standard modifier) the rest will inherit them so you won't need to include them in each toolbar type and you can use the modifier classes to overwrite these props. We always have a default behaviour (usually it's the standard type) and if we need to overwrite the default behaviour we rely on the modifier classes

Reply:
fixed